src/tech/promotion-process.md

```diff
@@ -4,343 +4,178 @@ redirect_from:
-  Technical Operations, and Technical Architects)
-* two Heads of Discipline, Principals, or Leads from outside the technology team
+* your Head of Discipline
+* your line manager
+* other members of the technology leadership team (Director of Technology, Head of Development, Head of Technical Operations, and Lead Technologists) and/or
+* Heads of Discipline, Principals, or Leads from outside the technology team
 
-The panel must not include your line manager, buddy, or anyone in a close
-personal relationship with you to avoid conflicts of interest. This includes
-people who would consider you a friend as well as a colleague.
+The panel will be as diverse as possible to get a mix of different perspectives and to reflect the makeup of the wider tech team. The specific makeup of the panel depends on things like people’s availability, the level of the role of the promotion, and any specific needs you have raised during the process.
 
-Your Head of Discipline appoints one member of the panel (not necessarily the
-most senior person and not your Head of Discipline) as the lead. It is their
-responsibility to organize the review process, run the joint panel, and make
-sure your application is properly considered.
+Panels evaluate applications based on the same Progression criteria for all applications for the same role, and will take advice from the People team on any reasonable adjustments required to ensure objectivity. The People team are also available to help take minutes and advise if necessary.
 
-Your promotion panel must meet **between 1 and 2 weeks after the application
-deadline** to be able to give you a timely decision and response while also
-considering your application properly.
+The names of the panel will be shared with you in advance. The panel should not include anyone with a conflict of interest. If this is unavoidable, then the People team should be involved.
 
-If there is more than one applicant for a single promotion opportunity, the
-panel must be different for each applicant. Some overlap between panels is ok if
-necessary, but should be avoided if possible.
+Your promotion panel should meet between 1 and 2 weeks after the application deadline to be able to give you a timely decision and response while also considering your application properly.
 
 ### Review process
 
-#### Independent review
+The panel will review the Progression check-ins and any accompanying docs. The review will focus on:
 
-**At least 1 week before the panel meets**, the lead panelist creates a review
-form from
-[a template](https://drive.google.com/drive/folders/1q4idxObiYhqOAWuzudFQcGAk_70yZk-I),
-transfers ownership of it to a member of the People team for filing, and sends it
-with your promotion pack to each member of the panel.
+Your current performance, primarily relating to the Progression framework, but also considering general professionalism and dxw values more generally
```

Comment:
I think this and the next two paras should be bullet points (they're indented in the Google Doc)